File: 001.py
# Completed on 4/29/2016 by Theresa Thoraldson
# source: https://projecteuler.net/problem=1
import math

dict = []

print("Euler 001")

for x in range(0, 1000):
  if (x % 5) == 0:
    dict.append(x)
  if (x % 3) == 0:
    dict.append(x)

print("list of factorials:"),
print list(set(dict))
print("")
print("Sum of factorials:"),
print sum(list(set(dict)))

File: javascript/001.js
/*Completed on 7/8/2016 by Theresa Thoraldson
  source: https://projecteuler.net/problem=1 */

var list = [];
 
for (i=1; i < 1000; i++) {
	if (i % 3 === 0) {
 		list.push(i);
 	}
 	else if (i % 5 === 0) {
 		list.push(i);
 	}
}

var count = 0;
 
for(var i=0, n=list.length; i < n; i++) {
   count += list[i]; 
}